DancingGoat/Views/Product/Brewer.cshtml

```diff
@@ -0,0 +1,10 @@
+﻿@model DancingGoat.Models.Brewer
+
+@{
+    Layout = "~/Views/Shared/_ProductLayout.cshtml";
+}
+
+@* The ProductSpecificProperties section is added just because of a small bug in MVC. MVC simply requires at least one of the declared sections to be implemented in the view. *@
```

Comment:
maybe it's just by design... please make the comment simple `MVC requires at least one of the declared sections to be implemented in the view.`